linux/jdk/debian/src/main/packaging/temurin/17/debian/changelog

```diff
@@ -1,3 +1,9 @@
+temurin-17-jdk (17.0.6.0.0+10) STABLE; urgency=medium
+
+  * Eclipse Temurin 17.0.6.0.0+10 release.
+
+ -- Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>  Wed, 18 Jan 2023 09:43:20 +0000
```

Comment:
think we usually use the date when the installer should be built not the one GA tag ready

Reply:
this date looks good to me, the GA date makes more sense as it allows users to compare against the tags